qemu/tests/cfg/x86_cpu_model.cfg

```diff
@@ -51,6 +85,20 @@
             only HostCpuVendor.intel
             flags = "split_lock_detect gfni movdiri movdiri64b cldemote umip"
             model_pattern = "Intel Atom Processor \(SnowRidge%s\)"
+        - Snowridge-v1:
```

Comment:
Do we need add guest restriction for some intel models as well?

Reply:
> Do we need add guest restriction for some intel models as well?

yes, will update later, thanks.

Reply:
@huangyum update, please help to review, thanks.